src/js/components/MesosLogView.js

```diff
@@ -281,7 +281,7 @@ class MesosLogView extends mixin(StoreMixin) {
 
     return (
       <pre
-        className="flex-grow flush-bottom"
+        className="flex-grow flush-bottom prettyprint"
```

Comment:
Yes, lee wrote this down in the notes above, so this shouldn't get lost.
Because this PR doesn't introduce the prettyprint itself, this should also be fixed outside of this PR for the sake of keeping the PRs small and consistent.